package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
why did u decide to create top level package?)

Reply:
I plan to use eslint globally for all tasks folders and therefore I added package.jsonfor main folder to use npm scripts globally for all tasks.